docs/CHANGELOG/changelog-0.1.x.md

```diff
@@ -0,0 +1,21 @@
+# v0.1.0
+
+The first official release of Claudie
+
+# Features
+
+- Multi-cloud kubernetes cluster management
+- Multi-cloud loadbalancer management
+- Fast scale-up/scale-down of defined infrastructure
+- Persistent storage via Longhorn
+- Support for AWS, Azure, GCP, OCI and Hetzner
+- Support for Cloud DNS on GCP only (for now)
+
+# Bugfixes
+
+- As this is first release there are no bugfixes
+
+# Known issues
+
+- `iptables` reset after reboot and block all traffic on `OCI` node #466
+- Numerous issues when deploying on `arm64` based machines
```

Comment:
we might be able to remove this if we wait for the PR that fixes this.

Reply:
I would rather remove it once its merged to master :)